bot/exts/evergreen/wtf_python.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import random
+import re
+from typing import Dict, Optional
+
+from discord import Embed
+from discord.ext import commands, tasks
+from fuzzywuzzy import process
+
+from bot import constants
+from bot.bot import Bot
+
+log = logging.getLogger(__name__)
+
+WTF_PYTHON_RAW_URL = ("http://raw.githubusercontent.com/satwikkansal/"
+                      "wtfpython/master/README.md")
+BASE_URL = "https://github.com/satwikkansal/wtfpython"
+THUMBNAIL = "https://raw.githubusercontent.com/satwikkansal/wtfpython/master/images/logo.png"
+
+ERROR_MESSAGE = f"""
+Unknown WTF Python Query. Please try to reformulate your query.
+
+**Examples**:
+```md
+{constants.Client.prefix}wtf wild imports
+{constants.Client.prefix}wtf subclass
+{constants.Client.prefix}wtf del
+```
+If the problem persists send a message in <#{constants.Channels.dev_contrib}>
+"""
+
+
+class WTFPython(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.get_wtf_python_readme.start()
+
+        self.data: dict = {}
+        self.headers: Dict[str] = dict()
+
+    @tasks.loop(hours=1)
+    async def get_wtf_python_readme(self) -> None:
+        """Gets content of README.md of WTF Python Repository."""
+        async with self.bot.http_session.get(WTF_PYTHON_RAW_URL) as resp:
+            if resp.status == 200:
+                self.raw = await resp.text()
+                await self.parse_readme(self.raw)
+            else:
+                log.debug(f"Failed to get latest WTF Python README.md. Status code {resp.status}")
+
+    async def parse_readme(self, data: str) -> None:
+        """
+        Parses the README.md into a dict.
+
+        Key is the name of the section and link to it.
```

Comment:
Did you rename `Key` to `data`?

Reply:
No, it is talking about how it parses the `README.md` into `self.headers`

Reply:
Ahh, so `self.headers` is a `name: link` dict?

Reply:
> Ahh, so `self.headers` is a `name: link` dict? If so, could you update this docstring to say that it parses the readme into the `self.headers` dict, where the key is the heading and the value is the link to the heading?

Yes you are right! Alright will update the `doc-string`.